### ISSUE TITLE
SP5: Axis ticks invisible when set AutoScaler.XInverted = true

### ISSUE THREAD

**Issue by mrchipset** [ScottPlot/ScottPlot#3379]
**Issue:** The Axis become invisible when set autoscale inverted axis

**ScottPlot Version:** 5.0.21

**Code Sample:**

```cs
var sig = Plot.Add.SignalXY(x, y);
sig.Color = Colors.Blue;
sig.Axes.YAxis = Plot.Axes.Right;
sig.Axes.XAxis = Plot.Axes.Bottom;
sig.Data.Rotated = true;
Plot.Axes.AutoScaler.InvertedX = true;
Plot.Axes.AutoScale();
```

![image](https://github.com/ScottPlot/ScottPlot/assets/36813506/438b8d8f-fc28-4fa2-86cc-347d4ff37001)

**Comment by BrianAtZetica** [comment]
This looks the same as my issue #3397 which I've resolved in pull request #3401

**Comment by swharden** [comment]
I assume this was resolved and will be in the next version of ScottPlot 5.0.22 

https://scottplot.net/changelog/

After the next release if this is still a problem we can re-open this issue and figure it out together 👍
